databases/mongodb-atlas/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "mongodb-atlas",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "devDependencies": {
+    "mongodb": "^3.6.6",
```

Comment:
So it seems we require mongo dep only to create an ObjectID like
```
  const ObjectID = require('mongodb').ObjectID;
  const objectId = new ObjectID().toString();
```

I think it's better if we can remove that dependency in the future

Reply:
Prisma currently does not offer this functionatliy afaik :/

Reply:
I guess not yet indeed